go.mod

```diff
@@ -0,0 +1,20 @@
+module github.com/Mellanox/ipoib-cni
+
+go 1.13
+
+require (
+	github.com/containernetworking/cni v0.7.1
+	github.com/containernetworking/plugins v0.8.3
+	github.com/coreos/go-iptables v0.4.2
```

Comment:
here there should be entries of projects that are directly used by ipoib-cni
i dont think this project uses all of these directly doesnt it ?

Reply:
sorry forgot to run go mod tidy